File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        "return an array where an index i of the array is the amount of days it takes for the temp to be greater" 
        out = []

        for i in range(len(temperatures)):
            j = 1
            while i + j < len(temperatures) and temperatures[i + j] <= temperatures[i]: 
                j += 1
            
            if i + j == len(temperatures):
                out.append(0)
            else:
                out.append(j)
        
        return out

File: Data Structures & Algorithms/reverse-string/submission-0.java
class Solution {
    public void reverseString(char[] s) {
        int front = 0;
        int back = s.length - 1;

        char temp;
        while (front < back && front < s.length && back > 0) {
            temp = s[front];
            s[front] = s[back];
            s[back] = temp;
            front++; back--;
        }
    }
}